### ISSUE TITLE
Dynamically show required star (*) based on requiredIf

### ISSUE THREAD

**Issue by ww99w** [vuelidate/vuelidate#1119]
Please export a property which computes the requiredIf as a boolean value.

I would like to use this property to dynamically set the CSS class

**Comment by victorgarciaesgi** [comment]
Yep, it can be really useful to have an active state of the validator to let us show a `*` or a required mark for the user experience
